tests/crud/sync_db/test_vector_tile_assets.py

```diff
@@ -98,3 +99,12 @@ def test_get_non_existing_fields():
 def test_latest_date():
     result = "2020-01-01"
     assert result == get_latest_date("nasa_viirs_fire_alerts", "v202003")
+
+
+def test_get_dataset_tile_caches():
```

Comment:
I mentioned above possibly breaking out your preview method into smaller helper functions. If you do that, could you also see if it's possible to add tests for those functions?